packages/venia-concept/src/components/ProductOptions/option.js

```diff
@@ -10,6 +10,20 @@ import defaultClasses from './option.css';
 const getItemKey = ({ value_index }) => value_index;
 
 class Option extends Component {
+
+    constructor(props) {
```

Comment:
How do you feel about rewriting this component as a function component? Look at some of the rewrites we've been doing lately for reference but basically we will convert things like state to `useState` and the creation of the values map to a `useMemo` generator.

Reply:
Will refactor to a function components @sirugh